src/manifest.ts

```diff
@@ -545,7 +550,12 @@ export class Manifest {
     // pull requests
     if (!this.separatePullRequests) {
       plugins.push(
-        new Merge(this.github, this.targetBranch, this.repositoryConfig)
+        new Merge(
+          this.github,
+          this.targetBranch,
+          this.repositoryConfig,
+          this.groupPullRequestTitlePattern
```

Comment:
I think passing the `manifestOptions` to all plugins would be cleaner, but in this `Manifest` constructor we don't capture the `manifestOptions` arg on this instance.